libimage/pull.go

```diff
@@ -201,15 +216,16 @@ func (r *Runtime) copyFromDefault(ctx context.Context, ref types.ImageReference,
 		if err != nil {
```

Comment:
Pre-existing: Shouldn’t `ociTransport` above have _exactly the same_ naming logic?

Reply:
That sounds like a good idea